src/components/NoteDialogComponents/NoteDialogNoteComponents/NoteDialogNoteContent.tsx

```diff
@@ -81,19 +86,30 @@ export const NoteDialogNoteContent: FC<NoteDialogNoteContentProps> = ({noteId, a
             )}
         </>
       ) : (
-        <textarea
-          className="note-dialog__note-content--text"
-          disabled={!editable}
-          onBlur={(e) => onEdit(noteId!, e.target.value ?? "")}
-          onFocus={onFocus}
-          onKeyDown={(e) => {
-            if (e.key === "Enter" && !e.shiftKey) {
-              e.preventDefault();
-              (e.target as HTMLTextAreaElement).blur();
-            }
-          }}
-          defaultValue={text}
-        />
+        <>
+          <textarea
+            className="note-dialog__note-content--text"
+            disabled={!editable}
+            onBlur={(e) => onEdit(noteId!, e.target.value ?? "")}
+            onFocus={onFocus}
+            {...emoji.inputBindings}
+            onKeyDown={(e) => {
+              emoji.inputBindings.onKeyDown(e);
+              if (e.defaultPrevented) return;
+
+              if (e.key === "Enter" && !e.shiftKey) {
+                e.preventDefault();
+                e.currentTarget.blur();
+              }
+            }}
+          />
+
+          {!isStackedNote && (
+            <div className="note-dialog__note-content--emoji-suggestions">
+              <EmojiSuggestions {...emoji.suggestionsProps} />
+            </div>
```

Comment:
Why did you choose to make the emoji dropdown only available for the top note?

Reply:
It looking somewhat weird having transparent suggestions overlap when you scroll

![grafik](https://github.com/inovex/scrumlr.io/assets/71148001/2890ad67-108a-46f9-8315-73e8ac755057)